### PR TITLE
{vis}[GCCcore/11.3.0] tesseract v5.3.0, Leptonica v1.83.0

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/tesseract/tesseract-5.3.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+modextrapaths = {
+    'TESSDATA_PREFIX': 'tessdata',
+}
+
+sanity_check_paths = {
+    'files': ['bin/tesseract'],
+    'dirs': ['share/tessdata', 'include/tesseract']
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/t/tesseract/tesseract-5.3.0-GCCcore-11.3.0.eb:62]: Could you please include one or more simple sanity check commands for these modules, to verify that the executables can actually be run (e.g. `tesseract --version` or something)?

Comment (crubb) [reply easybuild/easyconfigs/t/tesseract/tesseract-5.3.0-GCCcore-11.3.0.eb:62]: Done! Actually found, that I needed to correct `TESSDATA_PREFIX`, so thank you! ;)

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/b2045632cc748bb7449a6e0614d5bc08 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Pull request merged**